### DISCUSSION TITLE
Fix All template

### DISCUSSION THREAD

**Discussion by Bluscream** [super-linter/super-linter#6058] [General]
Does someone here already have a template set up that sets all `FIX_*` vars to true and pushes the fixed files automatically?

**Comment by ferrarimarco** [comment]
This might be something useful. Converting to an issue. Feel free to send a PR.

**Reply by Bluscream** [reply]
https://github.com/Bluscream/AMPTemplates/blob/dev/.github/workflows/lint-all.yml

**Reply by ferrarimarco** [reply]
Thanks for the example!

**Reply by ferrarimarco** [reply]
Added an example to the README in https://github.com/super-linter/super-linter/pull/6060 :)
